data/json/items/magazine/57.json

```
@@ -3,8 +3,17 @@
     "id": "fn57mag",
     "looks_like": "glock17_17",
     "type": "MAGAZINE",
-    "name": { "str": "Five-seveN 20-round magazine" },
-    "description": "A standard capacity magazine for use with the FN Five-seveN.",
+    "name": { "str": "5.7mm 20-round pistol magazine" },
+    "description": "A 20-round box magazine, for use with handguns chambering the proprietary 5.7x28mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "fn57mag",
+        "name": { "str": "Five-seveN 20-round magazine" },
+        "description": "A standard capacity magazine for use with the FN Five-seveN.",
```

```suggestion
        "description": "A standard-capacity magazine for use with the FN Five-seveN.",
```
Many New England states would consider this legally highcap(FN makes a ten rounder for those states). It is standard for the firearm tho